responsive-web-design/css-typography/index25.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Nutrition Label</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">
+  </head>
+  <style>
+    html {
+      box-sizing: border-box;
+    }
+
+    *, *:before, *:after {
+      box-sizing: inherit;
+    }
+
+    body {
+      font-family: 'Open Sans', sans-serif;
+      font-size: 16px;
+    }
+
+    .label {
+      border: 2px solid black;
+      width: 270px;
+      margin: 20px auto;
+      padding: 0 7px;
+    }
+
+    h1 {
+      font-weight: 800;
+      text-align: center;
+      margin: -4px 0;
+      letter-spacing: 0.15px;
+    }
+
+    p {
+      margin: 0;
+    }
+
+    .divider {
+      border-bottom: 1px solid #888989;
+      margin: 2px 0;
+    }
+
+    .bold {
+      font-weight: 800;
+    }
+  </style>
+  <body>
+    <div class="label">
+      <h1>Nutrition Facts</h1>
+      <div class="divider"></div>
+      <p>8 servings per container</p>
+      <p class="bold">Serving size 2/3 cup (55g)</p>
+    </div>
+  </body>
+</html>
+
+<!--
+Look at the CSS and you'll see that the code is not so DRY -- both the h1 and the new bold CSS rules have declarations that adjust the font weight.
```

Comment:
Define `DRY`

Reply:
I went back and defined `DRY` in `index.23.html`.